.circleci/config.yml

```diff
@@ -28,7 +28,21 @@ parameters:
   force-deploy-fxa-ci-images:
     type: boolean
     default: false
-
+  accounts-domain:
+    type: string
+    default: accounts.firefox.com
```

Comment:
@jbuck Should we default to GCP urls now?

Reply:
I think keeping it to the AWS prod URLs is fine - I don't think we can change the stage URLs easily, think about how many clients have them

Reply:
Hm, we should change this url to the stage URL or change the herokuapp domains to the prod versions

Reply:
I just looked at this again. I think we can leave them undefined, and then specify if/when we need to test a new URL. If these are pipeline parameters are not defined, the jobs will default to their typical settings, e.g. the defaults set in `functional-tests/lib/production.ts` and `functional-tests/lib/stage.ts`.